UEBPrep/app/views/cards/_form.html.erb

```diff
@@ -10,7 +10,31 @@
       </ul>
     </div>
   <% end %>
-
+    <% if @card.content_type_id == 1 %>
+        <h2>Text Content</h2>
+        <%= label_tag(:cardtext, "Text") %>
+        <%= text_area_tag( 'text', TextContent.find(@card.content_id).text , size: '8x8') %>
+    <% elsif @card.content_type_id == 2 %>
+        <h2>Card Title</h2>
+        <%= label_tag(:cardtitle, "Title") %>
+        <%= text_field_tag( 'title', TitledCardContent.find(@card.content_id).title) %>
+        <h2>Text Content</h2>
+        <%= label_tag(:cardtext, "Text") %>
+        <%= text_area_tag( 'text', TitledCardContent.find(@card.content_id).text , size: '8x8') %>
+    <% else %>
```

Comment:
Is ImageCardContent intended to be the catch-all? It might be safer to have an explicit elsif here and have the final else block display an error for unhandled card types.